app/components/TransferCrypto.tsx

```diff
@@ -0,0 +1,96 @@
+import { Textarea } from "@nextui-org/input";
+import { Button, Card, Link } from "@nextui-org/react";
+import { useState, useCallback, useEffect, useMemo } from "react";
+import { user_id } from "../../app_id/app_id.json";
+import { getSellTransactionStatus, createTransfer } from "../utils/queries";
+
+export default function TransferCryptoBox() {
+  const [amount, setAmount] = useState(0);
+  const [asset, setAsset] = useState("");
+  const [network, setNetwork] = useState("");
+  const [destination, setDestination] = useState("");
+  const [transfer_link, setTransferLink] = useState("");
+  const [transfer_result, setTransferResult] = useState("");
+
+  useEffect(() => {
+    const getOfframpTransaction = async () => {
+      const transaction = await getSellTransactionStatus({
+        partner_user_id: user_id,
+      });
+      try {
+        if (transaction) {
+          console.log(transaction);
```

Comment:
@wen-zhao-cb I think we should either remove the console logs, or give them a bit more context i.e. `console.log('[DEBUG] Got offramp transaction:', transaction);`

Reply:
I've cleaned up few console logs.